packages/backend/migrations/20210422194130_create_assets_table.js

```diff
@@ -13,6 +13,8 @@ exports.up = function (knex) {
     table.uuid('settlementBalanceId').notNullable()
     // TigerBeetle account id tracking reserved outgoing payments balance
     table.uuid('outgoingPaymentsBalanceId').notNullable()
+    // TigerBeetle account id tracking cumulative invoice receive limit
+    table.uuid('receiveLimitBalanceId').notNullable()
```

Comment:
@sentientwaffle can you explain the need for the extra balance here?

Reply:
Invoices have a new TigerBeetle account that track their progress towards the Invoice's receive limit. That way, the limit is enforced by TigerBeetle. https://github.com/interledger/rafiki/blob/825737fa98f3e9ed072319eac7640f6727b5846f/packages/backend/src/account/service.ts#L306-L314 This account on the asset counters the new one on the invoice/Account.

When the invoice is created, money is transferred from the invoice receive limit account to the asset's corresponding account. When payments to the invoice arrive, money is transferred the other direction. That way, Prepares which would put too much money in the invoice's receive limit account fail.